config/config_test.go

```diff
@@ -72,7 +72,23 @@ func TestGet(t *testing.T) {
 
 	config.Set("other::key1", "test key")
 
-	if v := config.String("multi::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+	if v := config.String("multi1::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "r.sub==p.sub&&r.obj==p.obj", v)
+	}
+
+	if v := config.String("multi2::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "r.sub==p.sub&&r.obj==p.obj", v)
+	}
+
+	if v := config.String("multi3::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "r.sub==p.sub&&r.obj==p.obj", v)
+	}
+
+	if v := config.String("multi4::name"); v != "" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "", v)
+	}
+
+	if v := config.String("multi5::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
```

Comment:
What's the difference between test ``multi1::name`` and ``multi5::name``?

Reply:
Multi5 contains an EOF while mutil1 not